Core/COObjectGraphContext.m

```diff
@@ -468,8 +486,11 @@ - (void)addItemsFromItemGraph: (id <COItemGraph>)itemGraph
 	// We could also change -[COObjectGraphContext itemForUUID:] to search
 	// itemGraph during the loading rather than the loaded objects (but that's
 	// roughly the same than we do currently).
-	_loadingItemGraph = itemGraph;
-	
+	// FIXME: Update loadableUUIDs when items are deleted in the migration.
```

Comment:
This comment can be removed imo. For itemUUIDs, we don't need to substract the UUIDs of the items deleted in the migration, since _loadingItemGraph is based on the migrated items and -mainItemsFromItemGraph:loadableUUIDs: will skip UUIDs not corresponding to any item in this graph.